### DISCUSSION TITLE
[A/L] (2021/22) Combinatoria - Esercizio 28

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#204] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152254369-4a5e4a22-54f4-4ebd-b816-1de9beb9ec86.png)

**Comment by Exyss** [comment]
1. (200 scelgo 4)
2. (100 scelgo 2)(100 scelgo 2)
3. Per complemento: #Tot - #SoloDispari = (200 scelgo 4) - (100 scelgo 4)

3b. Per tipizzazione diretta: (100 scelgo 1)(100 scelgo 3) + (100 scelgo 2)(100 scelgo 2) + (100 scelgo 3)(100 scelgo 1) + (100 scelgo 4)

Le due soluzioni sono equivalenti

**Comment by i0nut02** [comment]
confermo
